packages/react/src/atoms/textArea/base.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "../../../tailwind.css";
+import { useThemeContext } from "../../theme/ThemeProvider";
+import resolvedStyleProps from "../../utils/resolvedStyleProps";
+import LabelBase from "../labels/base";
+import pickAndMergeVariants from "../../utils/pickAndMergeVariants";
+
+export interface TextAreaBaseProps {
+  label?: any;
+  error?: any;
+  htmlFor?: string;
+  value?: string;
+  variant?: string;
+  placeholder?: string;
+  rows?: number;
+  cols?: number;
+  onChange?: (value?: string) => void;
+}
+
+const TextAreaBase = (props: TextAreaBaseProps) => {
+  const theme: any = useThemeContext();
+
+  const textAreaBaseClasses = pickAndMergeVariants(
+    "textAreaBaseClasses",
+    props,
+    theme
+  );
+
+  const textAreaClassNames = resolvedStyleProps(
+    "wrapper",
+    [
+      "display",
+      "border",
+      "background",
+      "width",
+      "color",
+      "font",
+      "padding",
+      "margin",
+    ],
+    props,
+    textAreaBaseClasses
+  );
+
+  return (
+    <>
+      {props.label !== "" && <LabelBase {...props} />}
+
+      <textarea
+        placeholder={props.placeholder}
+        className={textAreaClassNames}
```

Comment:
Check TextInput component once, there was some issue when label is not passed. Even when label is not there, it shows "m-1" margin, we shouldn't include margin when there is no label.

Reply:
@Subbu-betalectic Please check TextInput, how it's applied. In this one you just changed mt-1 to mb-1, but usually elements shouldn't be responsible what comes below it, they should accommodate what comes above it.